lib/ASTGen/Sources/ASTGen/LegacyParse.swift

```diff
@@ -0,0 +1,43 @@
+import ASTBridging
+import BasicBridging
+import SwiftSyntax
+import ParseBridging
+
+
+
+extension ASTGenVisitor {
+
+  private func bridgedSourceLocForParse(_ node: some SyntaxProtocol) -> BridgedSourceLoc {
+    return BridgedSourceLoc(at: node.position, in: self.base)
+  }
+
+  func generateWithLegacy(_ node: ExprSyntax) -> BridgedExpr {
+    // NOTE: Postfix expressions and sequence expressions share the same start
+    // location with the inner expression. This function must only be called on
+    // "top" expressions. Even if that expression is migrated.
```

Comment:
I’m not entirely sure what you mean by top expressions here. Maybe an example would help

Reply:
Rephrased.

Reply:
It’s a lot clearer now. Thanks!